cumulusci/utils/tests/test_option_parsing.py

```diff
@@ -154,3 +159,33 @@ def test_multiple_errors(self):
         assert "the_bool" in str(e.value)
         assert "req" in str(e.value)
         assert "Errors" in str(e.value)
+
+    def test_options_read_only(self):
+        # Has an Options class
+        task1 = TaskToTestTypes(self.project_config, self.task_config, self.org_config)
+        assert isinstance(task1.options, ReadOnlyOptions)
+        # Does not have an Options class
+        task2 = TaskWithoutOptions(
+            self.project_config, self.task_config, self.org_config
+        )
+        assert isinstance(task2.options, dict)
+
+    def test_init_options__options_read_only_error(self):
+        expected_error_message = "The 'options' dictionary is read-only. Please use 'parsed_options' instead."
```

Comment:
If the error message was a constant it could be imported and used here.

Reply:
Yep, makes more sense to use a constant